### ISSUE TITLE
用d3预训练模型训练shape，没有输出结果

### ISSUE THREAD

**Issue by Bourne-M** [zylo117/Yet-Another-EfficientDet-Pytorch#489]
训练：python train.py -c 3 -p shape --head_only True --lr 1e-3 --batch_size 42 --load_weights efficientdet-d3.pth  --num_epochs 50

测试：示例代码

**Comment by zylo117** [comment]
Do you mean the progression bar was progressing but no loss value was being printed?
Then it could be exploding gradients. Try a lighter network like D0 like I did.

**Comment by Bourne-M** [comment]
> Do you mean the progression bar was progressing but no loss value was being printed?
> Then it could be exploding gradients. Try a lighter network like D0 like I did.

不是的，好像cv2.rectangle(ori_imgs[i], (x1, y1), (x2, y2), (255, 255, 0), 2) 不行
需要先copy一份图像
img=ori_imgs[i].copy()
cv2.rectangle(img, (x1, y1), (x2, y2), (255, 255, 0), 2)
plt.imshow(imgs)

**Comment by zylo117** [comment]
I see. It's fixed. https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/commit/64eb1ac5de36e46b7bf80879c94480ea96c9f135. Sorry for the trouble.

**Comment by Bourne-M** [comment]
> I see. It's fixed. [64eb1ac](https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/commit/64eb1ac5de36e46b7bf80879c94480ea96c9f135). Sorry for the trouble.
可是我还是不能用自己的数据训练出可用的模型，用自己的数据训练的时候 loss已经很低了，但是测试的时候一直是：len(out[i]['rois']) 等于0

**Comment by Bourne-M** [comment]
在我的数据集上 效果很差，改用了tf版本的，效果要好很多
